include/wx/utils.h

```diff
@@ -780,6 +780,13 @@ void WXDLLIMPEXP_CORE wxGetMousePosition( int* x, int* y );
 
 #ifdef __WXGTK__
     WXDLLIMPEXP_CORE void *wxGetDisplay();
+    enum wxDisplayType
+    {
+        wxDisplayNone,
+        wxDisplayX11,
+        wxDisplayWayland
+    };
+    WXDLLIMPEXP_CORE void *wxGetDisplayAndType(wxDisplayType *type);
```

Comment:
It's a private function, so it's not too bad, but output parameters are still ugly. I'd rather have something like
```
struct wxDisplayInfo
{
    void* dpy;
    wxDisplayType type;
};
```
and return this struct instead.